src/pybind/mgr/dashboard/frontend/src/app/shared/forms/cd-validators.ts

```diff
@@ -238,4 +228,18 @@ export class CdValidators {
       );
     };
   }
+
+  static uuid(required = false): ValidatorFn {
```

Comment:
I guess you removed the description accidentally?

Reply:
But nice improvement ;)

Reply:
I've been so bold to leave out the type declarations as they are already covered by TypeScript. I hope that's okay.

Reply:
For the sake of documentation, consistency and completeness it should be kept here, but that's only my opinion.

Reply:
I've checked the documentation tools available and they all seem to support extraction of the types directly from the code, so personally, I see no reason to document and maintain it twice (as it's already documented in the code and covered by documentation generators).
For the sake of consistency, I'd remove the types in the doc strings if we conclude to leave them out (in a follow up PR).

Reply:
Okay, fine with me